README.md

```diff
@@ -17,19 +17,28 @@ Component-driven prototyping tool using [Pattern Lab v2](http://patternlab.io/)
   2. `cd emulsify`
   3. `npm install` or `yarn install`
 
+Or as a single command:
```

Comment:
We can probably get rid of the steps here, and just say "We support NPM and YARN" and then provide the two commands below. I don't think there's any value in having the steps broken out.